src/app/public/testing/infinite-scroll/infinite-scroll-fixture.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  DebugElement
+} from '@angular/core';
+
+import {
+  ComponentFixture
+} from '@angular/core/testing';
+
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+/**
+ * Provides information for and interaction with a SKY UX infinite scroll component.
+ * By using the fixture API, a test insulates itself against updates to the internals
+ * of a component, such as changing its DOM structure.
+ */
+export class SkyInfiniteScrollFixture {
+  private debugElement: DebugElement;
+
+  constructor(
+    private fixture: ComponentFixture<any>,
+    skyTestId: string
+  ) {
+    this.debugElement = SkyAppTestUtility.getDebugElementByTestId(fixture, skyTestId, 'sky-infinite-scroll');
+
+    this.waitForComponentInitialization();
+  }
+
+  public get loadMoreButtonIsVisible(): boolean {
+    return this.getButton() instanceof HTMLButtonElement;
+  }
+
+  public clickLoadMoreButton(): void {
```

Comment:
Because clicking the "Load More" button could rely on some asynchronous logic running in the background, consumers will most likely have to wait on it to finish before doing running their test expectations. To avoid that, we should probably make these `async` methods with `fixture.detectChanges()` and `return fixture.whenStable()`. Here's an [example](https://github.com/blackbaud/skyux-popovers/blob/master/src/app/public/testing/dropdown/dropdown-fixture.ts#L102).